src/utils/tools/updatingIcons/getSvgCode.ts

```diff
@@ -0,0 +1,28 @@
+import { ThemesDirectories } from 'common/constants'
+import { iconsTheme } from 'common/types'
+import fs from 'fs'
+
+const getFilled = async (iconName: string) => {
+  const iconPath = `${ThemesDirectories[iconsTheme.filled]}/${iconName}.svg`
+  if (fs.existsSync(iconPath)) {
+    const svgCode = fs.readFileSync(iconPath, 'utf8')
+    return svgCode
+  } else {
+    const outlinedSvgCode = await getOutlined(iconName)
+    return outlinedSvgCode
+  }
+}
```

Comment:
For the `filled` and `animated` icons, It will look for it first in the `svg` folder, if it's not available, it will look in `svg-outlined` folder.